index.bs

```diff
@@ -96,6 +96,25 @@ device running the user agent, and be accessed over a transport such as
 Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field
 Communications (NFC).
 
+## Specification Roadmap ## {#spec-roadmap}
+
+While many W3C specifications are directed primarily to audiences comprising user agent developers and web application developers (i.e., Web authors, herein also known as <em>[RP] developers</em>), the character of Web Authentication requires that this specification also be understood and properly used by those whose responsibilities include making decisions regarding:
+
+- [RP] web application login flows, account recovery flows, user account database content, etc. ([RP] developers)
+
+- Third-party web application frameworks / SDKs (Web framework developers)
+
+- OS platform API design and implementation in regards to platform-specific [=authenticator=] APIs, platform [=WebAuthn Client=] instantiation, etc. (OS platform developers)
+
+- [=Authenticator=] developers
+
+User agent developers, web framework developers, and OS platform developers ought to thoroughly understand the entire specification, except for perhaps the details of the various attestation statement formats ([[#defined-attestation-formats]]), since the latter are blobs emitted by authenticators that simply pass-through platforms and user agents on their way to the [RP] server (unless, of course, their platform emits an attestation statement of its own).
```

Comment:
Run-on sentence?

In general, it seems like each line here encourages each audience to understand the whole specification. In that case, breaking things down by audience seems unnecessary. Should we instead say something like "RP developers will want to start with the use cases and sample scenarios. Web authors will want to read the interface section carefully, and server developers will want to read the RP considerations section in detail... UA developers and web framework developers will want to start with the use cases and sample scenarios, and to read the interface section very carefully.... Authenticator developers will want to pay particular attention to the authenticator model and attestation sections..."? Also, could this be done in the respective bullet points above so we don't have two taxonomies of readers?